addons/hr_timesheet/views/project_sharing_views.xml

```diff
@@ -21,15 +21,14 @@
                     <group>
                         <group>
                             <div class="o_td_label">
```

Comment:
@IT-Ideas  changes done!
this div required to display labels on field (planned_hours) in a single line

Reply:
Ok thanks 🙂 
I had a check in community and indeed it looks ok. On the other hand `<field name="progress" widget="progressbar"/>` doesn't. I know it is not related to your diff, but it would be nice if you could set the same structure to that field too?
```js
<div class="o_td_label">
    <label for="x"/>
    <field name="x"/>
</>
```